sentry-android-core/src/main/java/io/sentry/android/core/internal/gestures/SentryGestureListener.java

```diff
@@ -186,6 +187,7 @@ private void addBreadcrumb(
 
   private void startTracing(final @NotNull UiElement target, final @NotNull String eventType) {
     if (!(options.isTracingEnabled() && options.isEnableUserInteractionTracing())) {
+      TracingUtils.startNewTrace(hub);
```

Comment:
also thinking if we should do to the same thing and not start a new trace if you're e.g. scrolling through the list?
```
    if (activeTransaction != null) {
      if (target.equals(uiElement)
          && eventType.equals(activeEventType)
          && !activeTransaction.isFinished()) {
                  options
            .getLogger()
            .log(
                SentryLevel.DEBUG,
                "The view with id: "
                    + viewIdentifier
                    + " already has an ongoing transaction assigned. Rescheduling finish")
```

Reply:
Updated